src/frontend/src/components/LongFormatButton/LongFormatButton.tsx

```diff
@@ -1,36 +1,40 @@
 // @ts-ignore
 import { ModalWrapper } from '@carbon/react';
-import { EmailNew } from '@carbon/icons-react';
 
 export type Props = {
     title: string;
     description: string;
+    icon: React.ReactNode;
+    passiveModal: boolean;
     buttonText: string;
     modalHeading: string;
     handleSubmit: Function;
     children: React.ReactNode;
 };
 
-export default function EmailModalButton({
+export default function LongFormatButton({
```

Comment:
made this generic for the app, it's actually a pretty neat icon + label + button component now